src/main/java/co/inventorsoft/academy/collections/model/Range.java

```diff
@@ -1,60 +1,83 @@
 package co.inventorsoft.academy.collections.model;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.Set;
+import java.util.*;
 
 public class Range<T> implements Set<T> {
 
+    private static List  mainList = new ArrayList();
+
     public int size() {
-        return 0;
+        return mainList.size();
     }
 
     public boolean isEmpty() {
-        return false;
+        return mainList.isEmpty();
     }
 
     public boolean contains(Object o) {
-        return false;
+        return mainList.contains(o);
     }
 
     public Iterator<T> iterator() {
-        return null;
+        return mainList.iterator();
     }
 
     public Object[] toArray() {
-        return new Object[0];
+        return mainList.toArray();
     }
 
     public <T1> T1[] toArray(T1[] a) {
         return null;
     }
 
     public boolean add(T t) {
-        return false;
+        return mainList.add(t);
     }
 
     public boolean remove(Object o) {
-        return false;
+        return mainList.remove(o);
     }
 
     public boolean containsAll(Collection<?> c) {
-        return false;
+        return mainList.containsAll(c);
     }
 
     public boolean addAll(Collection<? extends T> c) {
-        return false;
+        return mainList.addAll(c);
     }
 
     public boolean retainAll(Collection<?> c) {
-        return false;
+        return mainList.retainAll(c);
     }
 
     public boolean removeAll(Collection<?> c) {
-        return false;
+        return mainList.removeAll(c);
     }
 
     public void clear() {
+        mainList.clear();
+    }
+    public static Range of(int startList, int endList){
+        mainList.clear();
+        if (startList != endList) {
+            for (int i=startList; i<=endList; i++)
+            {
+                mainList.add(i);
+            }
+        }
+        return new Range();
 
     }
+
+    public static Range of(float startList, float endList){
+        mainList.clear();
+        if (startList != endList) {
+            for (float i=startList; i<=endList; i =i+0.1f)
+            {
```

Comment:
Keep one format for the code, open brackets in the same line, not in new one.
Format code, combination of ctrl + alt + L in idea should auto format it (at least in Linux it does)